uint64 timestamp		# timestamp this topic was emitted
float32 turn_distance		# the optimal distance to a waypoint to switch to the next
float32 landing_horizontal_slope_displacement
float32 landing_slope_angle_rad
float32 landing_flare_length
